packages/ee/billing/handlers/checkoutSessionCompleted.ts

```diff
@@ -0,0 +1,88 @@
+import { getTeam, updateTeam } from "@formbricks/lib/team/service";
+import { getMonthlyActivePeopleCount } from "@formbricks/lib/person/service";
+import { getMonthlyResponseCount } from "@formbricks/lib/response/service";
+import { getProducts } from "@formbricks/lib/product/service";
+
+import Stripe from "stripe";
+import { PriceLookupKeysInStripe, ProductFeatureKeysInDb, ProductNamesInStripe } from "../lib/constants";
+import { reportUsage } from "../lib/reportUsage";
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!, {
+  // https://github.com/stripe/stripe-node#configuration
+  apiVersion: "2023-10-16",
+});
+
+export const handleCheckoutSessionCompleted = async (event: Stripe.Event) => {
+  const checkoutSession = event.data.object as Stripe.Checkout.Session;
+  const stripeSubscriptionObject = await stripe.subscriptions.retrieve(
+    checkoutSession.subscription as string
+  );
+
+  const { customer: stripeCustomer } = (await stripe.checkout.sessions.retrieve(checkoutSession.id, {
+    expand: ["customer"],
+  })) as { customer: Stripe.Customer };
+
+  const team = await getTeam(stripeCustomer.metadata.team);
+  if (!team) throw new Error("Team not found.");
+  let updatedFeatures = team.billing.features;
+  let countForTeam = 0;
+
+  for (const item of stripeSubscriptionObject.items.data) {
+    const product = await stripe.products.retrieve(item.price.product as string);
+
+    switch (product.name) {
+      case ProductNamesInStripe.appSurvey:
+        updatedFeatures.appSurvey.status = "active";
+        if (item.price.lookup_key === PriceLookupKeysInStripe.appSurveyUnlimited) {
+          updatedFeatures.appSurvey.unlimited = true;
+        } else {
+          const products = await getProducts(team.id);
+          for (const product of products) {
```

Comment:
please use the new `getMonthlyTeamResponseCount(team.id)`

Reply:
done! used at other places as well and same for `getMonthlyActiveTeamPeopleCount`